bin/jobsub_fetchlog

```diff
@@ -51,6 +54,134 @@ from version import print_version, print_support_email
 
 verbose = 0
 
+# environment variable containing base fetchlog server url
+_FETCHLOG_URL_ENV = "JOBSUB_FETCHLOG_URL"
+# archive download chunk size in bytes
+_CHUNK_SIZE = 1024 * 1024
+
+
+def fetch_from_condor(
+    jobid: str, destdir: Optional[str], archive_format: str, partial: bool
+):
+    # find where the condor_transfer_data will put the output
+    j = Job(jobid)
+    iwd = j.get_attribute("SUBMIT_Iwd")
+    if verbose:
+        print(f"job output to {iwd}")
+    # make sure it exists, create if not
+    try:
+        os.stat(iwd)
+    except FileNotFoundError:
+        os.makedirs(iwd, mode=0o750)
+
+    # get the output sandbox
+    try:
+        j.transfer_data(partial)
+        transfer_complete = True
+    except htcondor.HTCondorIOError as e1:
+        print(f"Error in transfer_data(): {str(e1)}")
+    files = os.listdir(iwd)
+
+    if destdir is not None:
+        # If the user wants output in a specific directory, copy files there,
+        # don't build an archive. Old jobsub would get an archive from the
+        # server, upack it into the dest dir, then delete the archive.
+        owd = destdir
+        try:
+            os.stat(owd)
+        except FileNotFoundError:
+            os.makedirs(owd, mode=0o750)
+        for f in files:
+            shutil.copy2(os.path.join(iwd, f), owd)  # copy2 tries to preserve metadata
```

Comment:
At this point, do we want to remove the IWD?  Or because it's usually in ~/.cache, just let the OS clean it up whenever?